src/components/ha-fab.ts

```diff
@@ -0,0 +1,41 @@
+import { classMap, html } from "@material/mwc-base/base-element";
+import { ripple } from "@material/mwc-ripple/ripple-directive.js";
+
+import "@material/mwc-fab";
+// tslint:disable-next-line
+const MwcFab = customElements.get("mwc-fab");
+
+export class HaFab extends MwcFab {
+  protected render() {
```

Comment:
Can we add a comment with:
 - why we override the render method
 - what version of mwc-fab our render method was based on